index.html

```diff
@@ -69,7 +69,1116 @@
     </nav>
   </header>
   <main>
-
+    <section class="row align-items-start container-fluid mx-auto">
```

Comment:
## Improve the layout of the page.
- [ ] From a general flow. The page looks great but the layouts are not matching that of the source page. Let's compare the two pages below and work on the changes. The contents of the page appear to be centred and have spacing on both ends. There should be a bootstrap class to achieve this. Take a look at this [sandbox ](https://mdbootstrap.com/snippets/jquery/mdbootstrap/856950)and play with it then you can decide which way to go with this. I am open to discussion, just in case.  

|[Source  Page](https://web.archive.org/web/20210120125445/https://www.newsweek.com/)|[Clone Page](https://zilton7.github.io/newsweek-copy/)|
|:--:|:--:|
|![newsweek](https://user-images.githubusercontent.com/22345961/109695078-8fa4f800-7b9c-11eb-80cf-cf1a8cc506b5.png)|![newsweek-clone](https://user-images.githubusercontent.com/22345961/109695107-992e6000-7b9c-11eb-84b2-58db0964df8e.png)|